### PR TITLE
[BEAM-1375] Trigger builds on Github push notifications.

### PR DESCRIPTION
**Pull request by jasonkuster**
Signed-off-by: Jason Kuster <jasonkuster@google.com>

Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [x] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [x] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [x] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by jasonkuster** [comment]
R: @davorbonaci

File: .jenkins/common_job_properties.groovy [hunk 1]
```diff
@@ -158,14 +158,15 @@ class common_job_properties {
   // Sets common config for PostCommit jobs.
   static def setPostCommit(def context,
                            def build_schedule = '0 */6 * * *',
-                           def scm_schedule = '* * * * *',
+			   def poll_github = true,
```
Comment (davorbonaci) [inline .jenkins/common_job_properties.groovy]: spacing

Comment (davorbonaci) [inline .jenkins/common_job_properties.groovy]: `poll_github` -> `trigger_every_push`

File: .jenkins/job_beam_Release_NightlySnapshot.groovy [hunk 2]
```diff
@@ -34,11 +34,11 @@ mavenJob('beam_Release_NightlySnapshot') {
   common_job_properties.setMavenConfig(delegate)
 
   // Set that this is a PostCommit job.
-  // Polls SCM on Feb 31st, i.e. never.
+  // Does not poll SCM.
```
Comment (davorbonaci) [inline .jenkins/job_beam_Release_NightlySnapshot.groovy]: Merge these two lines:
`This is a post-commit job that runs once per day, not for every push.`

File: .jenkins/job_seed.groovy [hunk 3]
```diff
@@ -28,11 +28,11 @@ job('beam_SeedJob_Main') {
   common_job_properties.setTopLevelJobProperties(delegate)
 
   // Set that this is a PostCommit job.
-  // Polls SCM on Feb 31st, i.e. never.
+  // Does not poll SCM.
```
Comment (davorbonaci) [inline .jenkins/job_seed.groovy]: same here

File: .jenkins/common_job_properties.groovy (hunk 1 above)
Comment (jasonkuster) [inline .jenkins/common_job_properties.groovy]: Done

Comment (jasonkuster) [inline .jenkins/common_job_properties.groovy]: Done

File: .jenkins/job_beam_Release_NightlySnapshot.groovy (hunk 2 above)
Comment (jasonkuster) [inline .jenkins/job_beam_Release_NightlySnapshot.groovy]: Done.

File: .jenkins/job_seed.groovy (hunk 3 above)
Comment (jasonkuster) [inline .jenkins/job_seed.groovy]: Done

**Comment by jasonkuster** [comment]
Addressed comments

@davorbonaci PTAL

**Pull request closed without merging**